src/Bridge/Symfony/DependencyInjection/Compiler/DbToolsPass.php

```diff
@@ -29,7 +29,7 @@ public function process(ContainerBuilder $container): void
             }
 
             if (isset($config['anonymization']) && isset($config['anonymization']['yaml'])) {
-                foreach($config['anonymization']['yaml'] as $connectionName => $file) {
+                foreach ($config['anonymization']['yaml'] as $connectionName => $file) {
```

Comment:
I'm OK with all those changes, we'll merge with it, but please avoid polluting diffs with code style in the future. We do run cs fixer from time to time and make it explicit in commits.

Reply:
Here, it's probably a bit our fault: the [contribution rules](https://dbtoolsbundle.readthedocs.io/en/stable/contribute/guide.html) explicitly ask you to run `./dev.sh checks` before committing.

What's weird is that the CI shouldn't have accepted to merge these modifications that don't respect the code style. There must be a bug in the CI :shrug: